.drone.yml

```diff
@@ -71,22 +70,9 @@ platform:
 steps:
 - name: Build and Test
   commands:
-  - export SGX_MODE=HW
-  - export NETWORK_ID=ab
-  - . /opt/intel/sgxsdk/sgxsdk/environment
-  - rustup default nightly-2019-08-01-x86_64-unknown-linux-gnu
   - ls -l /dev/sgx
   - ls -l /var/run/aesmd/aesm.socket
-  - cd chain-tx-enclave/tx-validation && make clean && SGX_TEST=1 make
-  - cd bin && sh -c 'echo $$$ > $HOME/.PID; exec ./tx-validation-app'
-- name: Teardown
-  commands:
-  - kill "$(cat "$HOME/.PID")" || echo "./tx-validation-app already exited"
-  - (ps | grep ./tx-validation-app) || exit 0
-  when:
-    status:
-      - success
-      - failure
+  - docker run -v $HOME:/chain -ti --device /dev/sgx cryptocom/chain:latest /bin/bash /chain/ci-scripts/tx-validation-hw-test.sh
```

Comment:
I suggest adding `--rm` option to clean up the containers after finish

Reply:
Since we are not using the image type, perhaps we would want to add teardown step to bring the docker down.

Reply:
any quick idea for the teardown step? On a new machine, I may try to make them use the image type

Reply:
I think you can assign a random string docker `--name` and put it in a file (like `.PID` approach before), at teardown step you remove the docker base on the name in the file

Reply:
because ENV seems do not persist across steps, buf file does. So you have to use a file

Reply:
ok